packages/server/modules/webhooks/helpers/graphTypes.ts

```diff
@@ -0,0 +1,6 @@
+import { Webhook } from '@/modules/webhooks/domain/types'
+
+export type WebhookGraphQLReturn = Webhook & {
```

Comment:
this doesn't seem right - hasSecret & pprojectId don't need to be returned, cause they have their own resolvers. so probably `WebhookGraphQLReturn = Webhook` is enough